src/__acceptance_tests__/__ssr_pages__/main-navigation-bar-large.tsx

```diff
@@ -15,6 +15,7 @@ const NavigationBarTest = (): JSX.Element => (
             title,
             to: `/${title}`,
         }))}
+        logo={<span>LOGO</span>}
```

Comment:
SSR tests fail with Lazy components.
Ticket to review this: https://jira.tid.es/browse/WEB-1998